src/modules/admin/sagas/index.ts

```diff
@@ -75,7 +75,7 @@ function* colonyClaimToken({
 }
 
 function* colonyMintTokens({
-  payload: { amount, colonyAddress },
+  payload: { amount, colonyAddress, justification },
```

Comment:
Still need to record this somehow in here - can this/should this be wired as part of this PR, or separately?

Reply:
What you mean?